### ISSUE TITLE
The use of stashed tokens

### ISSUE THREAD

**Issue by Dingenis** [DinoChiesa/apigee-edge-js#7]
Hi,

How do I use the tokens that are stashed? Just connect without password, mfa_code?

```typescript
var options = {
      org: organization,
      user: identifier,
      no_token: false
    };
let org = await edge.connect(options);
```
When I do this, I get a missing password error because the stashed token is null. 

Cheers, 

Bram Sinke

**Comment by DinoChiesa** [comment]
Yes, if the stashed token is null, and if you do not provide login credentials, you will get an error at runtime. 

If the stashed token is non-null, apigee.connect() will succeed at runtime, and it will not use your login credentials.
